.github/workflows/conventional-pr-title.yaml

```diff
@@ -0,0 +1,18 @@
+name: 'PR title follows conventional commits'
```

Comment:
This is quite a long name, particularly with the limited space GitHub provides for it. It's definitely informative, and just seeing "PR title ..." is probably enough. But maybe worth shortening?